en/csql.rst

```diff
@@ -403,6 +403,12 @@ The **;SHELL_Cmd** command registers a shell command to execute with the **SHELL
     drwxr-xr-x  4 DBA cubrid   4096 Jul 29 16:14 cubridmanager
     csql>
 
+The OS environment variable(CUBRID_CSQL_SHELL, SHELL) may be registered and used in advance. ::
+  
+   $ export CUBRID_CSQL_SHELL='ls -la'
```

Comment:
I think the example is wrong.
Instead of entering a command such as ls -al, you must enter a shell command such as bash. (The default shell command is csh.)

Reply:
Looking at the explanation of SHELL_Cmd in the manual, I understood that it registered a command to be executed when the session command SHELL was entered. ([CSQL Session Commands manual](https://www.cubrid.org/manual/en/11.3/csql.html#session-commands))
Did I misunderstand something?

Reply:
What you understand is correct, but my opinion is changing the example so that it can be run with a shell command.

Reply:
The 11.3 manual also uses 'ls -la' as an example.
If I had to use another command, which one would seem appropriate?

Reply:
OK, just use the example as 'ls -al'.